src/index.ts

```diff
@@ -1,6 +1,9 @@
 import { Duplex } from 'stream';
 import { Runtime } from 'webextension-polyfill-ts';
 
+const browser: any  = globalThis.browser || globalThis.chrome;
```

Comment:
Assuming we want to just ignore these type errors, this would be one way:
```suggestion
const browser: any  = (globalThis as any).browser || (globalThis as any).chrome;
```

Not sure if there's a more concise way :thinking: We could also considering adding an interface for `globalThis` that included the stuff we use.